src/Api/Checkout.php

```diff
@@ -0,0 +1,162 @@
+<?php namespace Lean\Woocommerce\Api;
+
+use Lean\AbstractEndpoint;
+use Epoch2\HttpCodes;
+use Lean\Woocommerce\Utils\ErrorCodes;
+use Lean\Woocommerce\Utils\Hooks;
+
+/**
+ * Class Checkout.
+ *
+ * @package Lean\Woocommerce\Api
+ */
+class Checkout extends AbstractEndpoint
+{
```

Comment:
So we should same line brackets in classes also? I just did in that way cos I saw in one project and I followed that, and CI is not complaining haha.. 
```php
class Wathever extends Stuff {
}
```

Is that what you mean @mitogh ?